tensorflow/core/kernels/conv_grad_filter_ops.cc

```diff
@@ -1049,7 +1070,9 @@ REGISTER_KERNEL_BUILDER(Name("Conv2DBackpropFilter")
                               .TypeConstraint<T>("T")      \
                               .HostMemory("filter_sizes"), \
                           Conv2DSlowBackpropFilterOp<SYCLDevice, T>);
-TF_CALL_SYCL_NUMBER_TYPES(REGISTER_SYCL_KERNELS);
+
+//TF_CALL_SYCL_NUMBER_TYPES(REGISTER_SYCL_KERNELS)
```

Comment:
I believe we can un-comment that now

Reply:
If I remember correctly there was an issue with double taking too much space in some tests or some network and this was not handled correctly. I don't think it has been fixed (and probably won't be until we use SYCL-DNN). So for now we only use this convolution for float.

Reply:
Ah you are right. Lets leave it then!